app/docs/dev.md

```diff
@@ -1,29 +1,207 @@
+
+Each quantity should be written in the form `('quantity name','(units)')` and all pairs concatenated into a single string. See tables below for available default units.
+
+Whenever units are used they must be written exactly as in the left columns of the tables given below (no short forms or single-character symbols) and units must be multiplied (or divided) by each other, as well as any accompanying quantities. 
+
+To add further supported units, add their names to `names_of_prefixes_units_and_dimensions` in `buckingham_pi_utilities.py`.
+
+#### Table: Base SI units
+
+SI base units taken from Table 1 of the [NIST Guide to the SI, Chapter 4: The Two Classes of SI Units and the SI Prefixes](https://www.nist.gov/pml/special-publication-811/nist-guide-si-chapter-4-two-classes-si-units-and-si-prefixes)
+
+Note that gram is used as a base unit instead of kilogram.
+
+| SI base unit | Symbol | Dimension name      |
+|--------------|:-------|:--------------------|
+| metre        |   m    | length              |
+| gram         |   g    | mass                |
+| second       |   s    | time                |
+| ampere       |   A    | electriccurrent     |
+| kelvin       |   k    | temperature         |
+| mole         |  mol   | amountofsubstance   |
+| candela      |  cd    | luminousintensity   |
+
+#### Table: SI prefixes
+
+SI base units taken from Table 5 of [NIST Guide to the SI, Chapter 4: The Two Classes of SI Units and the SI Prefixes](https://www.nist.gov/pml/special-publication-811/nist-guide-si-chapter-4-two-classes-si-units-and-si-prefixes)
+
+| SI Prefix | Symbol | Factor     | | SI Prefix | Symbol | Factor     |
+|-----------|:-------|:-----------|-|-----------|:-------|:-----------|
+| quetta    |   Q    | $10^{30}$  | | deci      |   d    | $10^{-1}$  |
+| ronna     |   R    | $10^{27}$  | | centi     |   c    | $10^{-2}$  |
+| yotta     |   Y    | $10^{24}$  | | milli     |   m    | $10^{-3}$  |
+| zetta     |   Z    | $10^{21}$  | | micro     |   mu   | $10^{-6}$  |
+| exa'      |   E    | $10^{18}$  | | nano      |   n    | $10^{-9}$  |
+| peta      |   P    | $10^{15}$  | | pico      |   p    | $10^{-12}$ |
+| tera      |   T    | $10^{12}$  | | femto     |   f    | $10^{-15}$ |
+| giga      |   G    | $10^{9}$   | | atto      |   a    | $10^{-18}$ |
+| mega      |   M    | $10^{6}$   | | zepto     |   z    | $10^{-21}$ |
+| kilo      |   k    | $10^{3}$   | | yocto     |   y    | $10^{-24}$ |
+| hecto     |   h    | $10^{2}$   | | ronto     |   r    | $10^{-27}$ |
+| deka      |   da   | $10^{1}$   | | quecto    |   q    | $10^{-30}$ |
+
+#### Table: Common non-SI units
+
+Commonly used non-SI units taken from Table 6 and 7 of [NIST Guide to the SI, Chapter 5: Units Outside the SI](https://www.nist.gov/pml/special-publication-811/nist-guide-si-chapter-4-two-classes-si-units-and-si-prefixes)
 
-## Outputs
-*Output schema/values for this function*
+Note that the function treats angles, neper and bel as dimensionless values.
```

Comment:
Reformat as:

"Notes:
1. foo
2. bar"

Reply:
Removed note since units are no longer used for defining quantities